packages/frontend/src/app/[locale]/layout.tsx

```diff
@@ -34,6 +35,18 @@ export default async function Layout({
             <body>
                 <NextIntlClientProvider messages={messages}>
                     <ClientProviders locale={locale as Locale}>
+                        <Toaster
+                            duration={5000}
+                            visibleToasts={5}
+                            expand
+                            position="bottom-right"
+                            toastOptions={{
+                                unstyled: true,
+                                classNames: {
+                                    toast: "absolute right-0 rounded-2xl bg-white p-3",
```

Comment:
Can we have a CSS module class here?

Reply:
Oh right, fixed in 2eded801563eb6386c7f87da2e1264a3758b5b3e.